src/Npgsql/NpgsqlCommand.cs

```diff
@@ -169,7 +169,7 @@ internal static NpgsqlCommand CreateCachedCommand(NpgsqlConnection connection)
     /// <summary>
     /// Gets or sets the SQL statement or function (stored procedure) to execute at the data source.
     /// </summary>
-    /// <value>The Transact-SQL statement or stored procedure to execute. The default is an empty string.</value>
+    /// <value>The SQL statement or function (stored procedure) to execute. The default is an empty string.</value>
```

Comment:
Note #3913 where this is going to change, but for now that's a good change and consistent with the summary.